test/test_modules.py

```diff
@@ -0,0 +1,41 @@
+from copy import deepcopy
```

Comment:
to actually get this test to run in CI it has to be added to run_test.py

Reply:
True, do we want these running already? It's more likely people will look at them that way :p

Reply:
I think so -- otherwise the CI doesn't show they work